app/views/invite_mailer/send_invite.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>You have been invited to attend this special event: </h2>
+<h3>Dear <%= @user.first_name %> <%= @user.last_name %>, </h3>
+<p>
+  You have been invited to attend this special event:
+  <%= event.start_time %>
+  <%= event.duration %>
+  <%= event.description %>
+  <%= event.room.name %>
```

Comment:
this is placesaver code - we don't have a way now to segregate user that created event from user that is invited.